src/components/ImageSearch.vue

```diff
@@ -71,30 +94,78 @@
             },
             process_chunk(chunk) {
                 chunk.then((resp) => {
+
                     let resultArray = resp.data;
-                    // preprocess the flickr results:
-                    this.flickrImageItems = resultArray;
+                    this.flickrImageItems_buffer = resultArray;
+                    this.flickrImageItems = this.flickrImageItems.concat(this.flickrImageItems_buffer)
```

Comment:
Please don't do that, only load one bucket at a time. It is not necessary to load all buckets straight aways. That will end pretty bad when we will have all images processed